src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventProvider.cs

```diff
@@ -1288,8 +1266,120 @@ internal virtual int ActivityIdControl(Interop.Advapi32.ActivityControl ControlC
         {
             return IntPtr.Zero;
         }
-    }
 
-#pragma warning restore CA1852
+        protected unsafe void EnableCallback(
+                        EventProvider target,
+                        byte *additionalData,
+                        int controlCode,
+                        byte level,
+                        long matchAnyKeywords,
+                        long matchAllKeywords,
+                        Interop.Advapi32.EVENT_FILTER_DESCRIPTOR* filterData)
+        {
```

Comment:
The original code had a try {} catch {} surrounding this callback. Is that no longer needed?